src/containers/Secrets/Secrets.js

```diff
@@ -279,6 +303,8 @@ function mapStateToProps(state) {
     deleteSuccess: getDeleteSecretsSuccessMessage(state),
     loading: isFetchingSecrets(state),
     secrets: getSecrets(state),
+    fetchingSA: isFetchingServiceAccounts(state),
```

Comment:
only used with `loading`, could just combine them above and avoid adding an additional prop

`loading: isFetchingSecrets(state) || isFetchingServiceAccounts(state),`